gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -554,13 +554,16 @@ def prepare_ht_for_validation(
     :param vcf_info_reorder: Order of VCF INFO fields.
     :param joint_included: Whether joint frequency data is included in the HT. Default
         is False.
+    :param for_joint_validation: Whether to prepare HT for joint validation. Default is
+        True.
     :param freq_comparison_included: Whether frequency comparison data is included in
         the HT. Default is False.
     :return: Hail Table prepared for validity checks and export.
     """
     logger.info(
         "Unfurling nested gnomAD frequency annotations and add to INFO field..."
     )
+    for_joint = for_joint_validation
```

Comment:
Not here but I kept it in because it makes my next PR for the vcf export HT easier as I add an`for_joint_export` flag to the function and then add `or for_joint_export` to this line. I can remove it if you want though, I was being slightly lazy.

Reply:
Leave it then, the validate test could run. I think it can be merged.

Reply:
So I should merge this branch into yours?